src/pages/signin/ResendValidationForm.js

```diff
@@ -80,31 +85,58 @@ class ResendValidationForm extends React.Component {
     }
 
     render() {
+        const isNewAccount = !this.props.account.accountExists && !this.props.account.validated;
+        const isOldUnvalidatedAccount = this.props.account.accountExists && !this.props.account.validated;
+        const login = Str.isSMSLogin(this.props.credentials.login) ? this.props.toLocalPhone(Str.removeSMSDomain(this.props.credentials.login)) : this.props.credentials.login;
```

Comment:
nab - `Str.isSMSLogin(this.props.credentials.login)` called twice, could move to a var